test/integration/hook-err.spec.js

```diff
@@ -18,7 +18,34 @@ describe('hook error handling', function() {
   describe('before hook error tip', function() {
     before(run('hooks/before-hook-error-tip.fixture.js', onlyErrorTitle()));
     it('should verify results', function() {
-      assert.deepEqual(lines, ['1) spec 2', '"before all" hook:']);
+      assert.deepEqual(lines, [
```

Comment:
I'm not sure why (can't remember?) we're using `assert` in these tests, but please use `expect` like in the other tests.  if that fails for some reason, then we'll know why :smile:

You don't need to convert all tests in these files to use `expect` (unless you want to)

Reply:
@boneskull I'm trying to change "hook-err.spec.js" and to use `expect` as shown in f.e. "options.spec.js". As I understood so far, the key difference is: the "options.\*.fictures.js" don't use `console.log()` while "hooks.\*.fictures.js" do.
In order to use `expect` I need to call (helpers.js)runMochaJSON which results in a parsing error when using `console.log`. Probably I can change most of the tests accordingly, but there will be some depending on the `console.log` output.
Since JSON does not accept comments `//` I don't see a possibility to get around.
Do you have any tips?

Reply:
I think you figured this out?